File: src/test/java/cernx/colt/matrix/tdouble/impl/SparseDoubleMatrix2DViewTest.java
package cernx.colt.matrix.tdouble.impl;

public class SparseDoubleMatrix2DViewTest extends SparseDoubleMatrix2DTest {

    public SparseDoubleMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }
}

File: src/test/scala/cern/colt/matrix/tdouble/impl/SparseDoubleMatrix2DViewTest.scala
package cern.colt.matrix.tdouble.impl

import cern.colt.matrix.MatrixTypes.SparseDoubleMatrix2D

class SparseDoubleMatrix2DViewTest(arg0: String) extends SparseDoubleMatrix2DTest(arg0) {

  override protected def createMatrices() {
    val foo = new SparseDoubleMatrix2D(NCOLUMNS, NROWS)
    A = foo.viewTranspose()
    B = new SparseDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new SparseDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}
